.github/workflows/functional.yml

```diff
@@ -98,6 +98,14 @@ jobs:
             NGINX_CONF_DIR=internal/mode/static/nginx/conf
             BUILD_AGENT=gha
 
+      - name: Install cloud-provider-kind
+        run: |
+          curl -fsSL -O https://github.com/kubernetes-sigs/cloud-provider-kind/releases/download/v0.2.0/cloud-provider-kind_0.2.0_linux_amd64.tar.gz
```

Comment:
So this doesn't replace `kind` itself, it just adds the ability to create a load balancer?

Reply:
yeah it's a separate binary (for now?) that only adds a load balancer